src/components/Editor/Editor.js

```diff
@@ -30,21 +31,21 @@ export default function Editor() {
         <label>Alignment</label>
         <div className="radio-group">
           <label>
-            <input name="align" type="radio" value="left" />
+            <input name="align" type="radio" value="left" onChange={(e) => setAlign(e.target.value)} />
```

Comment:
Nice work Cole! If we wanted the option to display color on load, we can add a checked value => checked={align === 'left'} so that way the default value is viewed on page load. Excellent job!